### ISSUE TITLE
Getting a gauge critical error

### ISSUE THREAD

**Issue by sriv** [getgauge/gauge-visualstudio#114]
_From @sguptatw on May 17, 2016 5:34_

### Expected behavior
Should not get a gauge critical error, if it does I should get a link to report it to someone.

### Actual behavior
Getting a gauge critical error when a project dependency of gauge has some changes and can't report it.

### Steps to reproduce
1. Create a gauge project with a different solution folder.
2. Add a new project - `AProject`
3. Add dependency of the new project to the gauge project.
4. Make changes to `AProject` like adding a new method
```
public int getRandomNumber()
{
Random rnd = new Random();
int month = rnd.Next(1, 13); // creates a number between 1 and 12
int dice = rnd.Next(1, 7);   // creates a number between 1 and 6
int card = rnd.Next(52);    
return rnd;
}
```
5. Open a spec file
Got the following error.

<img width="359" alt="gauge critical error" src="https://cloud.githubusercontent.com/assets/6310197/15312034/4d80714e-1c1e-11e6-936e-263a7086af59.png">

### Gauge version
```
Gauge version: 0.4.1.nightly-2016-05-13

Plugins
-------
csharp (0.7.3.nightly-2016-05-13)
html-report (2.1.1.nightly-2016-05-03)
java (0.4.0)
ruby (0.2.0)
xml-report (0.1.2.nightly-2016-05-03)
```

_Copied from original issue: getgauge/gauge-csharp#70_

**Comment by sriv** [comment]
Issue is with Gauge-core, track at getgauge/gauge#405

**Comment by apoorvam** [comment]
Fixed in getgauge/gauge#405.

**Comment by apoorvam** [comment]
Fix should be available in nightly > 2016-05-25.
